app/main.ipynb

```diff
@@ -56,33 +58,59 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "# write your code here (for Python default syntax)\n",
+    "a = [1,2,3,4,5] * 1000\n",
+    "sys.getsizeof(a)"
    ],
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T12:02:41.673421Z",
-     "start_time": "2023-07-03T12:02:41.666277Z"
+     "end_time": "2024-10-24T15:39:46.380760Z",
+     "start_time": "2024-10-24T15:39:46.376681Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "40056"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 8
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "outputs": [],
    "source": [
-    "# write your code here (for numpy syntax)"
+    "# write your code here (for numpy syntax)\n",
+    "a = np.array(a, dtype=np.int8)\n",
+    "sys.getsizeof(a)"
```

Comment:
The numpy array `a` should be initialized with zeros and have a length of 1000, as per the task requirements. Consider using `np.zeros(1000, dtype=np.int8)` to create the array with minimal memory usage.